.vsts-ci.yml

```diff
@@ -1,3 +1,12 @@
+resources:
+  repositories:
```

Comment:
should arcade be the repository here?

Reply:
Yes, this adds a hook into Arcade for shared templates.  Kind of like the shared library concept from Jenkins.

https://github.com/Microsoft/vsts-agent/blob/master/docs/preview/yamlgettingstarted-reusefromrepo.md

Reply:
Ah, neat!

Reply:
With this hook, we could do things like, automatically add the MicroBuild plugin for you so that you don't have to, or add telemetry to your builds.

I'm debating whether teams will find value in automatically adding the MicroBuild plugin or if that's too much magic and teams would rather manage that themselves.

Telemetry for public VSTS builds is in development right now.